robot/Cumulus/tests/browser/Deepa.robot

```diff
@@ -0,0 +1,30 @@
+*** Settings ***
+Resource        robot/Cumulus/resources/NPSP.robot
+Library         cumulusci.robotframework.PageObjects
+...             robot/Cumulus/resources/ContactPageObject.py
+...             robot/Cumulus/resources/GiftEntryPageObject.py
+
+Suite Setup     Open Test Browser
+Suite Teardown  Close all browsers
+
+*** Variables ***
+${BROWSER}  chrome
```

Comment:
isn't chrome always our default browser? Why specify in a variable?

Reply:
this looks like deepas test file somehow got here, will delete this

Reply:
deleted file